src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +57,30 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
-
   return (
-    <>
-      <h4 className="admin-page-title">You are editing venue: {venue.name}</h4>
-      <BannerAdmin venueId={venueId} venue={venue} />
-      {isIframeVenue && <IframeAdmin venueId={venueId} venue={venue} />}
-    </>
+    <WithNavigationBar>
+      <div className="VenueAdminPage">
+        <h4 className="VenueAdminPage__title">
+          Current Announcement in Space Title
+        </h4>
+
+        {isBannerAdminVisibile ? (
+          <BannerAdmin
+            venueId={venueId}
+            venue={venue}
+            onClose={hideBannerAdmin}
+          />
+        ) : venue?.banner ? (
+          <>
+            <AnnouncementMessage banner={venue.banner} />
+
+            <AnnouncementOptions
+              banner={venue.banner}
+              onEdit={showBannerAdmin}
+            />
+          </>
+        ) : null}
```

Comment:
This whole chunk looks very complicated, Is it possible to split it into multiple conditions?
`{isBannerAdminVisibile....}`
`{!isBannerAdminVisibile....}`

Reply:
fixed 2431677